src/index.js

```diff
@@ -25,27 +25,61 @@ function setUrl(url, type='push') {
 }
 
 
+function getCurrentLocation() {
+	return (customHistory && customHistory.location) ||
```

Comment:
Depends - there was talk about moving `customHistory` to be a property of `Router` instances so that they had proper instance separation instead of the singleton stuff.  Might be better to go that route rather than making it more singelton-ey?  Different PR I guess though.